frappe/public/js/frappe/ui/dialog.js

```diff
@@ -252,6 +252,10 @@ frappe.ui.Dialog = class Dialog extends frappe.ui.FieldGroup {
 	}
 
 	hide() {
+		if (this.animate && this.animate === "slow") {
```

Comment:
"this.animate" is a boolean variable... better to add a new explicit variable. `animation_speed`? for the sake of better API.

Reply:
You are right. Overloading a boolean with a string is probably irritating, not smart…